templates/concert-ticket/concert-ticket.json

```diff
@@ -0,0 +1,44 @@
+{
+    "ticket_number": "000385724",
+    "event_time": "2021-05-25T19:30:00.00Z",
+    "ticket_detail": [
+        {
+            "field_name": "Admission",
+            "field_value": "General admission"
+        },
+        {
+            "field_name": "Ticket type",
+            "field_value": "Adult"
+        },
+        {
+            "field_name": "Price",
+            "field_value": "$32"
+        },
+        {
+            "field_name": "Name",
+            "field_value": "Coleman R. Merrell"
```

Comment:
Can you change the name to `Jane R. Doe`, just to be on the safe side?
Also, don't forget to regenerate the screenshot.